src/sdk/bucketspace_fs.js

```diff
@@ -96,8 +96,8 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
 
     async read_account_by_access_key({ access_key }) {
         try {
-            if (!access_key) throw new Error('no access key');
-            const iam_path = this._get_access_keys_config_path(access_key);
+            const iam_path = access_key === '' ? this._get_account_config_path(config.NC_ANON_ACCOUNT_NAME) :
```

Comment:
Using empty string or even undefined can be accidental and we really don't want to allow that.
Lets do this - define a new `const anonymous_access_key = Symbol('anonymous_access_key')` in object_sdk.
Then we should use this symbol as the key to the cache, but be explicit about it and not implicit.